unlock_indicator.c

```diff
@@ -272,13 +272,36 @@ xcb_pixmap_t draw_image(uint32_t *resolution) {
     }
 
     if (xr_screens > 0) {
+        /* This indicates if we have to draw the indicator to our current screen*/
+        bool draw_indicator = true;
+
+        /* This keeps track of the screens drawn to */
+        bool drawn_to[xr_screens];
```

Comment:
Strictly speaking this doesn't work if sizeof(bool) != 1, which is not required to be one, does it? For a multi-byte bool this might actually produce invalid bool-values and thus undefined behavior, no?